src/internal/components/option/index.tsx

```diff
@@ -61,6 +62,7 @@ const Option = ({
 
   return (
     <span
+      role={isPresentational ? 'presentation' : undefined}
```

Comment:
The presence of title and aria-disabled makes screen-readers consider the element as group which is unwanted when the group role is defined above. For token groups the group must include the dismiss button.

Reply:
I think having explicitly contradictory attributes like this is not a good thing, assuming they're already provided on the parent can't we remove them entirely?

Reply:
If we do not set role="presentation" the assumed role is "group" (at least in VO), probably because there is a title and aria-disabled available. That makes token include two groups - one is the LI item which includes the dismiss control and one is the option which does not include the dismiss.

Reply:
Sounds reasonable. How about I remove the role="presentation" part for now which can be justified by the fact the change improves a11y nevertheless? Then I can make the related changes as a follow-up. Wdyt?

Reply:
works for me!